src/cudd/adapter.h

```diff
@@ -362,6 +362,289 @@ class cudd_bcdd_adapter : public cudd_adapter
   }
 };
 
+class cudd_bdd_adapter : public cudd_adapter
```

Comment:
To keep a similar order, the `cudd_bdd_adapter` should probably preceede the `cudd_bcdd_adapter`? Or are you thinking lexical ordering?

Reply:
Fine by me, I just changed the order.